components/brave_rewards/resources/rewards_panel/components/vbat_notice_modal.style.ts

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Nit: Maybe we could update these to 2023.

Reply:
Done.